cran-comments.md

```diff
@@ -17,15 +17,14 @@ Package includes:
 * R-devel and release, on Win-builder
 
 * GitHub actions:
-  * Mac OS x 12.7.4 21H123, R 4.3.3
-  * Windows Server x64 2022, 10.0.20348, R 4.3.3
-  * Ubuntu 20.04.6 LTS, R-devel 2024-04-12
-  * Ubuntu 20.04.6 LTS, R 4.3.3
+  * Mac OS x 14.7.4 23H420, R 4.4.3
+  * Windows Server x64 2022 Datacentre, 10.0.20348, R 4.4.3
+  * Ubuntu 24.04.2 LTS, R-devel 2025-03-25
+  * Ubuntu 24.04.4 LTS, R 4.3.3
+  * Ubuntu 24.04.4 LTS, R 4.3.3
```

Comment:
Is this duplicated line intentional?  It may be - gh actions are like black magic!

Reply:
No, good spot.  One of them is 4.3.3 and one is 4.4.3. Will update prior to merge.  Ta!